charts/rancher-monitoring/100.2.0+up40.1.2/app-README.md

```diff
@@ -14,6 +14,20 @@ The chart installs the following components:
 
 For more information, review the Helm README of this chart.
 
+## Upgrading from 100.1.0+up19.0.3 to 100.2.0+up40.1.2
+
+### Rebased the chart to upstream version 40.1.2. 
+### Noticeable changes:
+
+Grafana: 
+- `grafana.serviceMonitor.enabled` default is set to `true`
```

Comment:
We always used to set this field to true, but it used to be known as `grafana.serviceMonitor.selfMonitor`: https://github.com/rancher/charts/blob/78fc6f2162c21af7025f504f8ed33f00bd0fe52c/charts/rancher-monitoring/101.0.0%2Bup19.0.3/templates/grafana/servicemonitor.yaml#L1

Don’t think this needs to be release noted here? What do you think?